core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -46,14 +46,15 @@ angular.module('oppia').directive('storyNodeEditor', [
         'ExplorationIdValidationService', 'StoryUpdateService',
         'UndoRedoService', 'EVENT_STORY_INITIALIZED',
         'EVENT_STORY_REINITIALIZED', 'EVENT_VIEW_STORY_NODE_EDITOR',
-        'AlertsService',
+        'AlertsService', 'MAX_CHARS_IN_CHAPTER_TITLE',
```

Comment:
Can you move the AlertServices on top (in sorted order)?

Reply:
Done